cfgov/agreements/requirements.txt

```diff
@@ -0,0 +1,2 @@
+tinys3==0.1.11
```

Comment:
Both of these requirements are already in `requirements/base.txt` (with a different version for `tinys3`). Can/should this file be removed?

Reply:
Yeah, it should be removed

Reply:
it's been removed